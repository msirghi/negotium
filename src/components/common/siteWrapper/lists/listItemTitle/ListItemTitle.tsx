import { FC } from 'react';
import { Typography } from '@mui/material';

type Props = {
  title: string;
};

export const ListItemTitle: FC<Props> = ({ title }) => {
  return <Typography sx={{ fontSize: 14 }}>{title}</Typography>;
};
