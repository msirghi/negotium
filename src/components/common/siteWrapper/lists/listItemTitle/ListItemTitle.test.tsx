import { ListItemTitle } from './ListItemTitle';
import renderer from 'react-test-renderer';

describe('ListItemTitle', () => {
  const defaultProps = {
    title: 'Title',
  };

  it('should match the snapshot', () => {
    const tree = renderer.create(<ListItemTitle {...defaultProps} />);
    expect(tree).toMatchSnapshot();
  });
});
