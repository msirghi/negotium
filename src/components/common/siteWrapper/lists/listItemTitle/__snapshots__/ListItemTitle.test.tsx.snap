// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemTitle should match the snapshot 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-1d9ye53-MuiTypography-root"
>
  Title
</p>
`;
